horace_core/sqw/@SQWDnDBase/SQWDnDBase.m

```diff
@@ -163,6 +163,5 @@
         status = adjust_aspect(w);
         [ok,mess,adjust,present]=adjust_aspect_option(args_in);
         dout = smooth_dnd(din, xunit, varargin);
-    end
+    end    
```

Comment:
looks like unncessary spaces added - accident? better to withdraw this commit?

Reply:
removed